pyanaconda/ui/gui/__init__.py

```diff
@@ -159,6 +159,15 @@ class in the action list instead of going on to
 
         self.builder.connect_signals(self)
 
+        # Hide keyboard indicator if we can't configure the keyboard
+        # It doesn't really give you any benefit of seeing something which could
+        # give you wrong values.
+        if (not conf.system.can_configure_keyboard) and \
+           isinstance(self.window, AnacondaWidgets.BaseWindow):
```

Comment:
This is sad, but I don't see a better option at this moment. Could you please specify in the comment, that this needs to be applied to every hub and spoke, but dialogs have to be skipped?

Reply:
Fixed